modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -1928,6 +1928,61 @@ definition {
 		ObjectAdmin.assertRequiredErrorPresent();
 	}
 
+	@description = "Verify if an error message appears when trying to save an Update an object Entry action with a Standalone trigger when using the invalid expression"
+	@priority = 4
+	test CannotSaveUpdateAnObjectEntryActions {
+		task ("Given: An object with a standalone action created") {
```

Comment:
This block is not adding the action yet. Please update to "Given: A custom object is created"